docker/Dockerfile

```diff
@@ -0,0 +1,110 @@
+FROM centos:latest
```

Comment:
Some other distributions are shipping hyperscan so this could result in a smaller docker image to use that instead of centos as the build prerequisites will be smaller.

Reply:
yes, like debian with backports which would bring both latest suricata and hyperscan versions.

In general I'm not quite sure why to pick one or the other as a base for docker. Guess image size is the main factor?

Reply:
Perhaps Ubuntu's new minimal image could be a good call. https://blog.ubuntu.com/2018/07/09/minimal-ubuntu-released

Reply:
Changed to the latest minimal Ubuntu Image and pushed. All tested good.